src/bp/core/app/botpress.ts

```diff
@@ -183,6 +183,13 @@ export class Botpress {
   private async maybeStartLocalSTAN() {
     const config = await this.moduleLoader.configReader.getGlobal('nlu')
 
+    if (!process.LOADED_MODULES['nlu']) {
+      this.logger.warn(
+        'NLU server is disabled. Enable the NLU module and restart Botpress to start the standalone NLU server'
```

Comment:
would have been nice to listen for NLU module being enabled, but **this is good enough** :+1: